src/org/exoplatform/shareextension/AccountsFragment.java

```diff
@@ -104,7 +104,7 @@ public boolean setViewValue(View view, Object data, String textRepresentation) {
   public void onListItemClick(ListView l, View v, int position, long id) {
     if (accounts != null && position >= 0 && position < accounts.size()) {
       ExoAccount acc = accounts.get(position);
-      if (acc.password != null && !"".equals(acc.password)) {
+      if (acc.isRememberEnabled && acc.isAutoLoginEnabled && acc.password != null && !"".equals(acc.password)) {
```

Comment:
I actually removed these checks before because the values are sometimes false even when the settings are enabled. This means the user always has to type his password. As I plan to refactor remember me and auto login, I preferred to ignore them here so the sharing operation is more user friendly.